config/settings.yml

```diff
@@ -74,6 +74,12 @@ email:
   general_from:
   exceptions_from:
   exceptions_to:
+  email_headers:
+    enabled: false
+    headers:
```

Comment:
The lines defining the headers should be commented out to show what they would look like but not shove in unnecessary headers into the config

Reply:
Okie doke. Thought that since they were set to false by default it would be okay, but that makes sense; smarter and safer